docs/reference/FAQ.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: FAQ
+parent: Reference
+nav_order: 4
+permalink: /reference/faq/
+---
+
+- TOC
+{:toc}
+
+# What languages does ClusterFuzz support?
+
+ClusterFuzz definitely supports C, C++ when compiled with
+[clang](https://clang.llvm.org/). It also supports Rust and it probably supports
+other languages that can be compiled with an LLVM-based toolchain (e.g. Swift).
+
+## Why is an LLVM-based toolchain needed for full support?
+
+An LLVM-based toolchain is needed for full ClusterFuzz support for two reasons:
+* Because ClusterFuzz relies on LLVM [sanitizers]({{ site.baseurl }}/reference/glossary/#Sanitizer) 
+  for detecting and identifying bugs.
+
+* Because use of [libFuzzer and AFL] in ClusterFuzz requires coverage
+  instrumentation only LLVM can provide.
+
+## What if I really don't want to use an LLVM-based toolchain?
+
+We strongly recommend using an LLVM-based toolchain. That said, you may be able
+to use some parts of ClusterFuzz with [GCC](https://gcc.gnu.org/) or extend
+ClusterFuzz to support your use of GCC. However, if you do this, you are in
+unexplored territory and we can not support this. GCC can probably be used for
```

Comment:
It's fine to leave this as-is, but it may be worth explicitly calling out some of the things we'd expect to break (deduplication, anything the relies on having reliable crashes) here to make it clearer to anyone who wants this support that they should try to use clang/sanitizers. I'm a bit torn since it seems to be explained above already, but I could see people being put off by this.

Reply:
I've never used asan with gcc, but do we expect the stack traces to be wrong?

Reply:
Landing this. I can change in new CL if you'd like.